lambda/smarthome/alexa/config.js

```diff
@@ -166,7 +166,7 @@ module.exports = Object.freeze({
       'category': 'SPEAKER',
       'properties': [
         {'name': 'muted', 'schema': 'muteState', 'isReportable': false, 'isSupported': false},
-        {'name': 'volume', 'schema': 'volumeLevel', 'isReportable': false, 'isSupported': false}
+        {'name': 'volume', 'schema': 'volumeIncrement', 'isReportable': false, 'isSupported': false}
```

Comment:
small nit pick, is this really a increment or should we refer to it as steps, so  'volumeSteps', those steps may increment or decrement it.

Reply:
I kept going back and worth 😄 I went with `increment` because I have used that term as metadata parameter, but I agree with you and will change it.